README.md

```diff
@@ -1,6 +1,6 @@
 # MassFuser: a workflow for combining MS1 and MS2 data from replicate LC-MS runs
 
-The idea behind MassFuser is to take advantage of the high resolution acquisition mode of modern mass spectrometers, and at the same time overcome its inherent limitation of longer instrument duty cycles. The principal use case of MassFuser is combining the data from replicate LC-MS runs, one of which is acquired in high resolution mode, giving accurate m/z values in MS1 scans, whereas the other one is processed with lower resolution mode and data independent MS/MS acquisition enabled. Thus, both LC-MS runs contain complementary information, which gets merged by MassFuser allowing for downstream analysis with any software tools accepting regular LC-MS data.
+The idea behind MassFuser is to take advantage of the high resolution acquisition mode of modern mass spectrometers, and at the same time overcome its inherent limitation of longer instrument duty cycles. The principal use case of MassFuser is combining the data from replicate LC-MS runs, one of which is acquired in high resolution mode, giving accurate m/z values in MS1 scans, whereas the other one is processed with lower resolution mode and data independent MS/MS acquisition enabled. Thus, both LC-MS runs contain complementary information, which gets merged by MassFuser allowing downstream processing with any open software tools accepting mzML format.
```

Comment:
@lfnothias jic you want your changes to be added to the repo, please update the README in your local branch as per our discussion, commit the changes, and I'll accept your pull request.